README.md

```diff
@@ -909,6 +909,8 @@ ssh -i YOUR_KEY_FILE \
 
 For a list of what is collected, see the logdog [command list](sources/logdog/src/log_request.rs).
 
+Log archives are also visible to host containers.
```

Comment:
Nit: move this line a few lines above, before "You can use SSH", and say something like:

```
This archive is accessible from host containers at `<blah>`.
....
```